frigate/api/defs/review_query_parameters.py

```diff
@@ -1,28 +1,31 @@
-from typing import Optional
+from typing import Union
 
 from pydantic import BaseModel
+from pydantic.json_schema import SkipJsonSchema
+
+from frigate.review.maintainer import SeverityEnum
 
 
 class ReviewQueryParams(BaseModel):
-    cameras: Optional[str] = "all"
```

Comment:
Why was optional removed?

Reply:
Because it would generate wrong documentation with `Optional` (https://github.com/pydantic/pydantic/pull/6653)

This 
```py
cameras: str = "all"
```

It's still optional, with a default value `all`. It will generate the following OpenAPI documentation:

```yaml
 - name: cameras
          in: query
          required: false
          schema:
            type: string
            default: all
            title: Cameras
```

The old code 

```py
cameras: Optional[str] = "all"
```

would generate wrong documentation like:

```yaml
    - name: cameras
          in: query
          required: false
          schema:
            anyOf:
              - type: string
              - type: 'null'
            default: all
            title: Cameras
```

The solutions are:

```py
cameras: str = "all"
```
or
```py
cameras: Union[str, SkipJsonSchema[str]] = "all"
```

For properties that can have the value `None`, then we need the `SkipJsonSchema`

```py
propertyName: Union[<type>, SkipJsonSchema[None]] = None
```

Reply:
gotcha, thanks for the explanation